ferrocene/doc/safety-manual/src/rustfmt/constraints.rst

```diff
@@ -0,0 +1,46 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+.. default-domain:: qualification
+
+Constraints
+===========
+
+Install version control system
+------------------------------
+
+.. id:: RUSTFMT_CSTR_0010_HAVE_VCS
+
+Associated requirement ID: :id:`RUSTFMT_AVD_CHECK_DIFF_001`
+
+Before executing ``rustfmt``, the user shall ensure that the a version control
+system (VCS, e.g. ``git``) is installed and used.
+
+Review diff
+-----------
+
+.. id:: RUSTFMT_CSTR_0020_USE_VCS
+
+Associated requirement ID: :id:`RUSTFMT_AVD_CHECK_DIFF_001`
+
+After executing ``rustfmt``, the user shall review, with the help of the VCS, the
+modifications ``rustfmt`` performed and ensure that there are no semantic changes.
```

Comment:
Should we mandate the use of a VCS at all? I would hope that our customers use one, but if they want to use a different technique to review the changes they should be free to.

Reply:
I think we should not mandate VCS, especially since `rustfmt` works fine without it

Reply:
Like, we do want to mandate that they manually review the changes done by rustfmt did not cause any difference in the semantic meaning of the code, but we shouldn't say *how* they do it.